lib/OpenQA/WebAPI/Controller/API/V1/Table.pm

```diff
@@ -352,6 +353,23 @@ Use by both B<create()> and B<update()> method.
 sub _prepare_settings {
     my ($self, $table, $entry) = @_;
     my $validation = $self->validation;
+    my $hp;
+    # accept both traditional application/x-www-form-urlencoded parameters
+    # with hash entries having key names encoded like settings[value1]
+    # (see doc at the end of HashedParams.pm)
+    # as well as modern application/json encoded hashes
+    if ($self->req->headers->content_type =~ /^application\/json/) {
```

Comment:
It would make sense to store the return value of `$self->req` in a variable instead of invoking this function over and over again (especially because it is even used in a loop).

Reply:
This is not in a loop.

Reply:
I see a loop below using `$self->req`.